packages/widgets/src/dockpanel.ts

```diff
@@ -1010,6 +1029,14 @@ namespace DockPanel {
      * If not given, default values will be used.
      */
     edges?: IEdges;
+
+    /**
+     * Allow tabs to be movable by user.
+     * When disabled the user will be unable to customize the layout.
```

Comment:
I would remove the `When disabled` line ... because presumably resizing the `TabPanel` instances still counts as customizing the layout.